src/main/java/core/basesyntax/Figure.java

```diff
@@ -15,7 +15,8 @@ public void setFigureColor(String figureColor) {
         this.figureColor = figureColor;
     }
 
-    public abstract double obtainArea();
+    @Override
+    public abstract double getArea();
 
-    public abstract void drawFigure();
+    public abstract void draw();
```

Comment:
[checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#dont-use-abstract-classes-to-set-behaviour-for-classes)